### DISCUSSION TITLE
App Access through Tunnel

### DISCUSSION THREAD

**Discussion by erasma** [gravitational/teleport#9092] [General]
I am trying to create app access - my device is connected to the teleport server through the tunnel which is working fine I am getting the following error 

running: on the node
teleport app start \
   --token= \
   --ca-pin= \
   --auth-server=serverip:443 \
   --name=appname \
   --uri=http://127.0.0.1:3001/

![image](https://user-images.githubusercontent.com/16279991/142873747-5ed02a7e-6570-4c5f-916f-41c8370c61e5.png)

**Comment by webvictim** [comment]
That looks like an issue with loading AWS credentials from the environment on the node. Could you please raise a bug report (https://github.com/gravitational/teleport/issues/new?assignees=&labels=bug&template=bug_report.md) with all requested details? This will help us track it down and fix it.

**Reply by erasma** [reply]
Thank you I have reported the bug

**Reply by webvictim** [reply]
Bug report is here for future reference: https://github.com/gravitational/teleport/issues/9095
